.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐛 Bug report
+description: Create a report to help us reproduce and fix a bug
+labels: [bug]
+
+body:
+- type: textarea
+  attributes:
+    label: Description
+    description: >
+      Please provide a clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps to reproduce
+    description: >
+      We can't solve your issue if we can't reproduce it. Please provide a
+      [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example)
+      that shows how to reproduce the bug. If the bug requires any additional files to
+      reproduce, please upload those or provide a download link. Your code should be
+      runnable and include all relevant imports.
+    placeholder: |
+      1. Download any Landsat 8 scene from https://earthexplorer.usgs.gov/
+      2. Run the following code to reproduce the error
+
+      ```python
+      from torchgeo.datasets import Landsat8
+
+      ds = Landsat8("/path/to/downloads/directory")
+      ...
+      ```
+  validations:
+    required: true
+- type: input
+  attributes:
+    label: Version
+    description: |
+      What version of TorchGeo are you using? This can be found using the following code.
+      ```console
+      $ python
+      >>> import torchgeo
+      >>> torchgeo.__version__
+      X.Y.Z
+      ```
+      If you are using a development release (ends with ".dev0") please also include the specific git commit.
+
+      Whenever possible, try to reproduce your issue with the latest commit from `main`. You never know, someone may have already fixed your bug!
+    placeholder: |
+      1.2.3.dev0 (e1285e6cc6b65080e82bdbf7de9dea3f647d8b3b)
+  validations:
+    required: true
+- type: markdown
+  attributes:
+    value: >
+      Thanks for taking the time to report this bug! TorchGeo is an open-source project
+      maintained by its users. If you're Python savvy and want to contribute a pull
+      request to fix this bug, we'll be happy to review it. If not, we'll try to fix it
+      as long as we can reproduce it.
```

Comment:
Wanted to clarify that TorchGeo isn't a commercial product with a team of developers just waiting to fix your bugs lol